de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/externallyreferenced/ExternallyReferencedCandidate.java

```diff
@@ -180,6 +185,7 @@ private ExternallyReferencedCandidate(
 		this.taxId = taxId;
 		this.invoiceDocTypeId = invoiceDocTypeId;
 		this.lineDescription = lineDescription;
+		this.invoiceDetailItems = invoiceDetailItems;
```

Comment:
this.invoiceDetailItems = invoiceDetailItems != null ? ImmutableList.copyOf(invoiceDetailItems) : ImmutableList.of();

Reply:
done